Jasmine/Swapping/ViewModels/BaseSwappingViewModel.swift

```diff
@@ -1,47 +1,9 @@
 import Foundation
 
-class BaseSwappingViewModel: SwappingViewModelProtocol {
-    /// Stores the grid data that will be used to display in the view controller.
-    private(set) var gridData: TextGrid
-    /// Number of rows in the grid, according to the answers property
-    var numRows: Int {
-        return gridData.numRows
-    }
-    /// Number of columns in the grid, according to the answers property
-    var numColumns: Int {
-        return gridData.numColumns
-    }
+class BaseSwappingViewModel: GridViewModel, SwappingViewModelProtocol {
```

Comment:
Prefer you conform to the Protocol with an extension, so can partition code clearer